ramses_rf/const.py

```diff
@@ -193,3 +193,25 @@
     "357": "Appliance in air purge mode. Primary heat exchanger air venting program active - approximately 100 seconds.",
     "358": "Three way valve kick. If the 3-way valve hasn't moved in within 48 hours, the valve will operate once to prevent seizure",
 }
+
+# used by 31DA
+EXHAUST_FAN_SPEED   = "exhaust_fan_speed"
+BOOST_TIMER         = "remaining_time"
+REMAINING_TIME      = "remaining_time"
+CO2_LEVEL           = "co2_level"
+INDOOR_HUMIDITY     = "indoor_humidity"
+AIR_QUALITY         = "air_quality"
+AIR_QUALITY_BASE    = "air_quality_base"
+OUTDOOR_HUMIDITY    = "outdoor_humidity"
+EXHAUST_TEMPERATURE = "exhaust_temperature"
+SUPPLY_TEMPERATURE  = "supply_temperature"
+SPEED_CAP           = "speed_cap"
+BYPASS_POS          = "bypass_pos"
+FAN_INFO            = "fan_info"
+SUPPLY_FAN_SPEED    = "supply_fan_speed"
+POST_HEAT           = "post_heat"
+PRE_HEAT            = "pre_heat"
+SUPPLY_FLOW         = "supply_flow"
+EXHAUST_FLOW        = "exhaust_flow"
+OUTDOOR_TEMPERATURE = "outdoor_temperature"
+INDOOR_TEMPERATURE = "indoor_temperature"
```

Comment:
Also, please edit **parser.py**, so that the `31D9`/`31DA` parsers are using the above constants too:
```python
@parser_decorator  # ventilation state extended, HVAC
def parser_31da(payload, msg) -> Optional[dict]:
  ...
    return {
        "exhaust_fan_speed": percent(payload[38:40]),  # 31D9[4:6]
        "remaining_time": double(payload[42:46]),  # mins, 22F3[2:6]
        "co2_level": double(payload[6:10]),  # ppm, 1298[2:6]
        "indoor_humidity": percent(payload[10:12], high_res=False),  # 12A0?
        "air_quality": percent(payload[2:4]),
        "air_quality_base": int(payload[4:6], 16),  # 12C8[4:6]
        "outdoor_humidity": percent(payload[12:14], high_res=False),
        "exhaust_temperature": double(payload[14:18], factor=100),
        "supply_temperature": double(payload[18:22], factor=100),
        "indoor_temperature": double(payload[22:26], factor=100),
        "outdoor_temperature": double(payload[26:30], factor=100),  # 1290?
        "speed_cap": int(payload[30:34], 16),
        "bypass_pos": percent(payload[34:36]),
        "fan_info": CODE_31DA_FAN_INFO[int(payload[36:38], 16) & 0x1F],
        "supply_fan_speed": percent(payload[40:42]),
        "post_heat": percent(payload[46:48], high_res=False),
        "pre_heat": percent(payload[48:50], high_res=False),
        "supply_flow": double(payload[50:54], factor=100),  # L/sec
        "exhaust_flow": double(payload[54:58], factor=100),  # L/sec
    }
    ```